cmd/kots/cli/install.go

```diff
@@ -614,3 +644,66 @@ func getHttpProxyEnv(v *viper.Viper) map[string]string {
 	return env
 
 }
+
+func isAppReady(namespace string) string {
+
+	log := logger.NewLogger()
+	stopCh := make(chan struct{})
+	defer close(stopCh)
+
+	clientset, err := k8sutil.GetClientset(kubernetesConfigFlags)
+	if err != nil {
+		return ""
+	}
+	podName, err := k8sutil.FindKotsadm(clientset, namespace)
+	if err != nil {
+		return ""
+	}
+
+	localPort, errChan, err := k8sutil.PortForward(kubernetesConfigFlags, 0, 3000, namespace, podName, false, stopCh, log)
```

Comment:
i suggest this logic be moved to the `pollApplicationStatus` and then pass the required variables as arguments, because this will do a port forward on every check (which is currently every 3 seconds).

Reply:
yes makes sense.